app/dashboard/admin.py

```diff
@@ -327,6 +330,12 @@ class HackathonSponsorAdmin(admin.ModelAdmin):
     list_display = ['pk', 'hackathon', 'sponsor', 'sponsor_type']
 
 
+class HackathonWorkshopAdmin(admin.ModelAdmin):
+    """The admin object for the HackathonWorkshop model."""
+
+    list_display = ['pk', 'start_date', 'hackathon', 'speaker', 'url']
```

Comment:
let's change here speaker to raw_id_fields.

Reply:
Fixed @octavioamu